bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/discovery/BusGatewayUpnpDiscovery.java

```diff
@@ -226,7 +226,7 @@ private DeviceInfo(RemoteDevice device) {
                     }
                 }
             }
-            logger.info("Found BTicino device: not a OpenWebNet gateway or is not supported (UDN={})", idString);
+            logger.info("Found BTicino device: not a OpenWebNet gateway or not supported (UDN={})", idString);
```

Comment:
Even this one could be rather DEBUG? 
We should have the minimum of INFO logs.

Reply:
I would find useful to tell the user via INFO message that the BTicino device that was found is NOT a gateway that can be used by this binding for integration. Therefore I would leave the level to INFO.

Reply:
I let another maintainer decide.
@kaikreuzer ?